todo/src/reducers/index.js

```diff
@@ -0,0 +1,27 @@
+
+
+const initialState = {
+    todos : [
+        {
+            id: 1, 
+            item: "create reducer",
+            completed: false
+        },
+        {
+            id: 2,
+            item: "create actions",
+            completed: false
+        }
+    ]
+}
+
+
+function reducer(state=initialState, action){
+    switch(action.type){
+        default:
```

Comment:
You're going to need to add a case: ADD_TODO which creates a new To-do object. It should look something like this: 

```suggestion
  export default (state = initialState, action) => {
  switch (action.type) {
    case "HANDLE_INPUT":
      return Object.assign({}, state, { input: action.payload });
    case "ADD_TODO":
      if (state.input === "") {
        return Object.assign({}, state, { todos: state.todos, input: "" });
      }
      const nextId = state.todos.length + 1;
      let nextTodo = { id: nextId, value: state.input, completed: false };
      let addTodos = state.todos.slice();
      addTodos.push(nextTodo);
      return Object.assign({}, state, { todos: addTodos, input: "" });

    case "TOGGLE_TODO":
      let toggleTodos = state.todos.slice();
      toggleTodos.map(todo => {
        if (todo.id === action.payload) {
          todo.completed = !todo.completed;
          return todo;
        }
        return todo;
      });
      return Object.assign({}, state, { todos: toggleTodos });

    default:
      return state;
  }
};

```